.cspell.json

```diff
@@ -702,6 +703,7 @@
     "angular.json",
     "ng-package.json",
     "libs/shared/src/types/timezones/timezones.types.ts",
-    "*.riv"
+    "*.riv",
+    "cafebabe"
```

Comment:
@SokratisVidros i think it should be added at the top section of words, vs the file paths key here

Reply:
Fixed.